src/frontend/App/utils/utils.ts

```diff
@@ -59,7 +59,7 @@ export const utledBehandlingLenke = (
     behandling: Behandling,
     eksternLenker: Eksternlenker
 ): string => {
-    return `${utledFagsystemUrl(behandling.fagsystem, eksternLenker)}/fagsak/${
+    return `${utledFagsystemUrl(behandling.fagsystem, eksternLenker)}/ekstern/fagsak/${
```

Comment:
Dette blir bare riktig for EF - så du må legge `/ekstern` i miljøvariablene for ef-lenker

Reply:
For barnetrygd bruker ikke konseptet "ekstern" - da blir det smell.. 💥